### ISSUE TITLE
Missing images in About Us page

### ISSUE THREAD

**Issue by espinielli** [ATFutures/hugo-site#4]
It looks like your template needs images for every `.md` files in `content/about-us`.
The YAML for each has a `image: ""` but this is a problem, see snapshot below:

![Screen Shot 2020-10-26 at 16 49 11](https://user-images.githubusercontent.com/891692/97195105-3b7fa680-17ab-11eb-9e42-f03bb76cc142.png)

Checking [the theme you use](https://github.com/MarcusVirg/forty/blob/07cbe7a93693aedd8df606c964153f5afecb419a/layouts/_default/list.html#L17), maybe you just need to remove `image: ""`

**Comment by Robinlovelace** [comment]
:+1: Thanks for the heads-up @espinielli.

**Comment by Robinlovelace** [comment]
Any ideas @mpadge ?

**Comment by mpadge** [comment]
Yeah, idea is to rebuild the whole site anyway, coz that one is crap, and hugo is way more complicated than a static generator needs to be. I'll take this as impetus to do something about it asap. Thanks both!
